grunt/tasks/eslint.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var grunt = require('grunt');
+
+module.exports = function() {
+  var done = this.async();
+  grunt.util.spawn({
+    cmd: 'node_modules/.bin/eslint',
+    args: ['src/', 'Gruntfile.js', 'grunt/', 'main.js', 'perf/', 'test/']
```

Comment:
@hzoo this doesn't work as well.
BTW `**/*.js` works. But 
```
➜  react git:(remove-jshint) ./node_modules/eslint/bin/eslint.js **/*.js
zsh: argument list too long: ./node_modules/eslint/bin/eslint.js
```
so we need more flexible grunt task or maybe some grunt plugin. But I hope this can be another PR because this diff is too large already.